pkg/ceph/client/status.go

```diff
@@ -59,7 +59,9 @@ type HealthStatus struct {
 
 type CheckMessage struct {
 	Severity string `json:"severity"`
-	Message  string `json:"message"`
+	Summary  struct {
```

Comment:
Can you extract out this Summary struct? I havent seen nested structs declared this way.